PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSoftDropResponse.h

```diff
@@ -68,6 +72,17 @@ class AliAnalysisTaskSoftDropResponse : public AliJetResponseMaker {
   Int_t                       fZ2gAxis;                                 ///< add Z2g axis in matching THnSparse (default=0)
 
  private:
+ 
+  void               Decluster(const fastjet::PseudoJet& jet);
+  fastjet::ClusterSequence* Recluster(const AliEmcalJet* jet);
+  
+  void               FillZgRgVectors(const AliEmcalJet* jet);
+
+  Int_t nsdsteps;
+  std::vector<Float_t> zg_values; //n_SD ordering
+  std::vector<Float_t> rg_values; //n_SD ordering
+  std::vector<Int_t>   SDsteps_values;
```

Comment:
You should:
- update your ClassDef
- rename your variables to start with "f" to keep with convention
- Set the streaming properties (ie `//!<!` or `///<`)

Reply:
According to the discussion in #5173, you don't need to update your ClassDef if these are just transient. But I think the other comments are still important

Reply:
Pardon my ignorance, where the new streamer instructions are explained? I find only old style //!

Reply:
I probably should have included a link. See [here](https://dberzano.github.io/alice/doxygen/#data_members)